pkg/gen/python-templates/yaml.py.mustache

```diff
@@ -3,7 +3,7 @@
 
 import json
 from copy import deepcopy
-from typing import Optional
+from typing import (Dict, List, Optional)
```

Comment:
I don't think you need parens here: `from typing import Dict, List, Optional` should work.